.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 build
 *.log
-package-lock.json
```

Comment:
Was this part intentional? I thought that for libraries, we don't usually check this in.

It's probably good to use a specific version of `tree-sitter-cli`, but we can already just do that via the package.json, since it is a dev dependency.

Reply:
ah yeah I thought it would be good to use `npm ci` which I think works a bit better with caching and needs the package-lock.json to be checked in, but I just realized there isn't caching set up for node_modules (and it probably doesn't make sense to do so since the dependency tree is so small)

:+1: